src/bicep/examples/newWorkload/newWorkload.bicep

```diff
@@ -67,32 +125,32 @@ module spokeNetwork '../../modules/spokeNetwork.bicep' = {
   scope: az.resourceGroup(resourceGroup.name)
   params: {
     tags: calculatedTags
-
-    logStorageAccountName: logStorageAccountName
+    location:location    
+    logStorageAccountName: workloadLogStorageAccountName
     logStorageSkuName: logStorageSkuName
 
     logAnalyticsWorkspaceResourceId: logAnalyticsWorkspaceResourceId
 
     firewallPrivateIPAddress: firewallPrivateIPAddress
 
-    virtualNetworkName: virtualNetworkName
+    virtualNetworkName: workloadVirtualNetworkName
     virtualNetworkAddressPrefix: virtualNetworkAddressPrefix
     virtualNetworkDiagnosticsLogs: virtualNetworkDiagnosticsLogs
     virtualNetworkDiagnosticsMetrics: virtualNetworkDiagnosticsMetrics
 
-    networkSecurityGroupName: networkSecurityGroupName
+    networkSecurityGroupName: workloadNetworkSecurityGroupName
     networkSecurityGroupRules: networkSecurityGroupRules
     networkSecurityGroupDiagnosticsLogs: networkSecurityGroupDiagnosticsLogs
     networkSecurityGroupDiagnosticsMetrics: networkSecurityGroupDiagnosticsMetrics
 
-    subnetName: subnetName
+    subnetName: workloadSubnetName
     subnetAddressPrefix: subnetAddressPrefix
     subnetServiceEndpoints: subnetServiceEndpoints
   }
 }
 
 module workloadVirtualNetworkPeerings '../../modules/spokeNetworkPeering.bicep' = {
-  name: take('${workloadName}--VNetPeerings', 64)
+  name: take('${workloadName}-to-hub-vnet-peering', 64)
```

Comment:
I updated this to match the convention used in `hubNetworkPeerings.bicep`.  Let me know if these are changes are correct by resolving this conversation.

https://github.com/Azure/missionlz/blob/199b986c770d0d331b6d99fce39d9b9807af2e28/src/bicep/modules/hubNetworkPeerings.bicep#L5